synapse/handlers/receipts.py

```diff
@@ -139,13 +142,17 @@ async def received_client_receipt(
         if not is_new:
             return
 
+        if not self.hs.should_send_read_receipts():
+            return
+
```

Comment:
I think `_handle_new_receipts` returns `False` in this case, so this is unreachable and redundant?

Reply:
Yep, think you're correct.

Reply:
I think you're right...